target/linux/mvebu/modules.mk

```diff
@@ -17,6 +17,27 @@ endef
 $(eval $(call KernelPackage,linkstation-poweroff))
 
 
+define KernelPackage/turris-omnia-mcu
+  SUBMENU:=$(OTHER_MENU)
+  TITLE:=CZ.NIC's Turris Omnia MCU driver
+  DEPENDS:=@TARGET_mvebu_cortexa9 @TESTING_KERNEL
```

Comment:
Please use `@!LINUX_5_15`instead of `@TESTING_KERNEL`. 

We could miss this when we change mvebu to kernel 6.1 by default. when you use not 5.15 we will find it when we remove kernel 5.15 support and can remove the dependency.

Reply:
I backported for 5.15 as well, so no need to this. Will push once I test it.